### ISSUE TITLE
add empty "alt" attributes to "div.profileImage" > img in "Profile" objects: 0.5

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#332]
At present, no "alt" attributes appear on the "profileImage" ```<img>``` elements that appear in our custom "Profile" content type. This is a WCAG A accessibility violation. E.g.:

```html
<div class="ProfileImage">
    <img src="http://isaw.nyu.edu/people/faculty/isaw-faculty/lorenzo-dalfonso/@@images/Image/mini">
</div>
```

But what to put in the ```alt``` attribute? [W3C guidance](https://www.w3.org/WAI/tutorials/images/decorative/) would have us distinguish between "informative" and "decorative" images, with the latter deserving an empty ```alt``` attribute, effectively muting it in screen readers. We think that staff photos would be informative to a sighted person, but not informative to a person experiencing the site through a screen reader; therefore, we will treat these images in the "People Listing" as decorative.

Thus, we should modify the appropriate template to produce:

```html
<div class="ProfileImage">
    <img alt="" src="http://isaw.nyu.edu/people/faculty/isaw-faculty/lorenzo-dalfonso/@@images/Image/mini">            
</div> 
```

**Comment by paregorios** [comment]
Looks good. Please merge to master and deploy to production unless it's tangled up in #335
